package.json

```diff
@@ -48,16 +48,25 @@
     "sinon": "^1.14.1"
   },
   "dependencies": {
+    "animate.css": "^3.2.5",
+    "bootbox": "^4.4.0",
+    "bootstrap": "^3.3.5",
+    "bootstrap-notify": "^3.1.3",
+    "classnames": "2.1.2",
```

Comment:
classnames must be fixed at 2.1.2 since ```var ClassNames = require('classnames'); will return undefined in current version 2.1.3, please see also https://github.com/JedWatson/classnames/issues/40